### ISSUE TITLE
Nuxt compatibility

### ISSUE THREAD

**Issue by bovas85** [cristijora/vue2-transitions#3]
I managed to get this plugin to work on nuxt, but I do get a barebones <component> in the html which isn't ideal.

Is there any recommended way to load this for SSR?

```
[Vue warn]: Unknown custom element: <component> - did you register the component correctly? For recursive components, make sure to provide the "name" option.

found in

---> <TransitionGroup>
       <FadeTransition>
         <NoSsr>
           <Pages/storesAndServices/index.vue> at pages/stores-and-services/index.vue
             <Nuxt> at .nuxt/components/nuxt.vue
               <Default> at layouts/default.vue
                 <Root>
...
```

**Comment by cristijora** [comment]
Hi @bovas85 Will look into that since I will have to use this in nuxt myself.

**Comment by bovas85** [comment]
Cheers @cristijora

**Comment by cristijora** [comment]
Released a new version. If you can give feedback on that @bovas85 would be awesome

**Comment by bovas85** [comment]
It seems fine @cristijora , although I can't override the tag.

Some people would probably need this to be a customisable tag (like for transition-group being a ul)

**Comment by cristijora** [comment]
@bovas85 Could you please tell what version of Vue are you using ?
Each transition component is using `$attrs` and `$listeners` which should make it possible to override any prop from `transition/transition-group` components. 
This feature unfortunately is only available in vue 2.4 and above

**Comment by cristijora** [comment]
Just to be sure, added a tag prop to components. Should work now. 
This is available in a new version [0.2.2](https://github.com/cristijora/vue2-transitions/releases/tag/v0.2.2)

**Comment by bovas85** [comment]
I'll test it tomorrow. I have 2.4 Vue on Nuxt latest
